schemas/chosen.yaml

```diff
@@ -52,6 +52,21 @@ properties:
       the Linux EFI stub (which will populate the property itself, using
       EFI_RNG_PROTOCOL).
 
+  rng-seed:
+    allOf:
+      - $ref: types.yaml#definitions/uint8-array
```

Comment:
Don't need the allOf here unless you have some more constraints. Is there a min or max length?

Reply:
Length depends on what bootloader can provide, but kernel won't check the length. Suggested min length can be 64 byte. In this case should min be added?